packages/fury-adapter-apib-parser/test/validate-test.js

```diff
@@ -53,7 +54,7 @@ describe('API Blueprint validation', () => {
                         content: [
                           {
                             element: 'number',
-                            content: 0,
+                            content: 11,
```

Comment:
NOTE, Added FORMAT: 1A so we're detecting document as API Blueprint and thus that is 11 additional characters at the start.